### PR TITLE
Autoscaling az

### PR DESCRIPTION
**Pull request by resouer**
Should be able to support this feature.

### CODE REVIEW

File: src/ClusterBootstrap/template/kubeconfig.yaml.template [hunk 1]
```diff
@@ -0,0 +1,18 @@
+apiVersion: v1
```
Comment (hongzhili) [inline src/ClusterBootstrap/template/kubeconfig.yaml.template]: Is this a copy of "src/ClusterBootstrap/template/kubelet/worker-kubeconfig.yaml.template" ?

Comment (resouer) [reply src/ClusterBootstrap/template/kubeconfig.yaml.template]: Not really, we recently changed cluster scaler to local usage only so the cert path will be different

Comment (hongzhili) [reply src/ClusterBootstrap/template/kubeconfig.yaml.template]: can you create a sub-folder under src/ClusterBootstrap/template and put this file to the sub-folder?

Comment (resouer) [reply src/ClusterBootstrap/template/kubeconfig.yaml.template]: yes!

Comment (resouer) [reply src/ClusterBootstrap/template/kubeconfig.yaml.template]: Fixed! Thanks!

**Pull request merged**